NuRadioMC/utilities/Veff.py

```diff
@@ -256,6 +273,11 @@ def get_Veff_Aeff_single(filename, trigger_names, trigger_names_dict, trigger_co
         logger.warning(f"file {filename} has no triggering events. Using trigger names from a different file: {trigger_names}")
 
     weights = np.array(fin['weights'])
+    # if zenith range is restricted, multiply with zenith mask inside bounds
+    if theta_fraction < 1:
+        mask_theta = np.array((fin['zeniths']>thetamin) & (fin['zeniths']<thetamax))
```

Comment:
Zenith refers here to the key of a file, right? Maybe add a comment, that this is on event level

Reply:
done!